### PR TITLE
Fix/toolbox wss redirect

### PR DESCRIPTION
**Pull request by MacondoExpress**
# Description

Provide a way to enable/disable encryption to Neo4j

### CODE REVIEW

File: packages/graphql-toolbox/src/contexts/auth.tsx [hunk 1]
```diff
@@ -86,7 +87,8 @@ export function AuthProvider(props: any) {
     [value, setValue] = useState<State>({
         login: async (options: LoginOptions) => {
             const auth = neo4j.auth.basic(options.username, options.password);
-            const driver = neo4j.driver(options.url, auth);
+            const encrypted = options.secure === "on" ? 'ENCRYPTION_ON' : 'ENCRYPTION_OFF';
```
Comment (tbwiss) [inline packages/graphql-toolbox/src/contexts/auth.tsx:90]: Nitpicking of the month: Let's also use double quotes for `ENCRYPTION_ON` and `ENCRYPTION_OFF`

**Review by tbwiss — DISMISSED** [review]
Thank you @MacondoExpress!
Really great addition!

**Comment by tbwiss** [comment]
@MacondoExpress, as you probably have seen, the tests seem to be failing. Maybe check the `login` function of the `Login` page, there could be a change needed.

File: packages/graphql-toolbox/src/contexts/auth.tsx (hunk 1 above)
Comment (MacondoExpress) [reply packages/graphql-toolbox/src/contexts/auth.tsx:90]: You know that I will not survive without your nitpickings :)

**Comment by tbwiss** [comment]
@MacondoExpress great! Let's merge this PR.

**Pull request merged**